README.md

```diff
@@ -12,7 +22,7 @@ To generate config you can use `rails generate couchbase_orm:config`:
 It will generate this `config/couchbase.yml` for you:
 
     common: &common
-      hosts: localhost
+      connection_string: couchbase://localhost
```

Comment:
why not hosts?

Reply:
It's the way of configurating the couchbase-ruby-client gem : https://github.com/couchbase/couchbase-ruby-client/blob/main/lib/rails/generators/couchbase/config/templates/couchbase.yml